tests/by-util/test_head.rs

```diff
@@ -378,3 +378,48 @@ fn test_presume_input_pipe_5_chars() {
         .run()
         .stdout_is_fixture("lorem_ipsum_5_chars.expected");
 }
+
+#[cfg(all(not(target_os = "windows"), not(target_os = "macos")))]
+fn run_and_check_if_it_outputs_something(ts: &TestScenario, args: &[&str]) {
```

Comment:
no need for this function, please just inject this into the various tests

Reply:
done